Interface.C

```diff
@@ -93,12 +93,41 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh)
 
             // Assign the (x,y,z) locations to the vertices
             for (int i = 0; i < faceCenters.size(); i++)
-            {
-                vertices[verticesIndex++] = faceCenters[i].x();
-                vertices[verticesIndex++] = faceCenters[i].y();
-                if(dim_ == 3)//out-of-plane axis = z
-                    vertices[verticesIndex++] = faceCenters[i].z();
+              for (unsigned int d = 0; d < dim_; ++d)
+                vertices[verticesIndex++] = faceCenters[i][d];
+
+#ifdef ADAPTER_DEBUG_MODE
```

Comment:
@MakisH Not sure if this corresponds only to messages or general DEBUG as well.

Reply:
`ADAPTER_DEBUG_MODE` is set in https://github.com/precice/openfoam-adapter/blob/8ae5f061ab26ae059b8cdcf304089306f9ce7227/Allwmake#L20 and yes, it also refers to general debug mode as well.